CompoundV3-Bots/Reserves-Monitor/src/agent.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  BlockEvent,
+  Finding,
+  HandleBlock,
+  FindingSeverity,
+  FindingType,
+  getEthersProvider,
+} from "forta-agent";
+import CONFIG from "./agent.config";
+import { BigNumber, providers } from "ethers";
+import Fetcher from "./dataFetcher";
+import { NetworkManager } from "forta-agent-tools";
+import { NetworkData, abs } from "./utils";
+import { createFinding } from "./finding";
+
+const ALERTS: { [address: string]: number } = {};
```

Comment:
I think it's a lot better to use a state object than a global variable like this, especially with the workarounds that are needed when testing. Check my PR for an `AgentState` example, it's a lot better to deal with than this - with it, you'll be able to read/write the state fields in the tests.